### PR TITLE
DOCS-2015: Remove rtsp model from RDK

### PR DESCRIPTION
**Pull request by sguequierre**
* Rework this rtsp readme for more Viam configuration information

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,30 +1,92 @@
...
+
+This module implements the [`"rdk:component:camera"` API](https://docs.viam.com/components/camera/) in an `rstp` model.
+With this model, you can configure a streaming camera with an h264 or h265 track for use with Viam.
+
+> [!NOTE]
+> For more information, see [Modular Resources](https://docs.viam.com/registry/#modular-resources).
+>
+> If your streaming camera uses the `x264` streaming video format, use the [`rtsp-h264` camera](https://app.viam.com/module/erh/viamrtsp) instead.
+
+## Requirements
```
Comment (sguequierre) [inline README.md:11]: @seanavery Question that always confuses me with module repos: Are these requirements being executed as part the module setup for those configuring from registry (so I could move to "Local Setup" section), or are they a must-do for module usage even if you are configuring from registry?

**Comment by randhid** [comment]
> * Rework this rtsp readme for more Viam configuration information
 please make a pull request here, we're going to deled this repo
https://github.com/erh/viamrtsp

**Comment by sguequierre** [comment]
> > * Rework this rtsp readme for more Viam configuration information
> >   please make a pull request here, we're going to deled this repo
> >   https://github.com/erh/viamrtsp

Oops! Was not aware. No problem

**Comment by seanavery** [comment]
@sguequierre Sorry about the confusion with this repo. It was used for POC work but the official repository is indeed at https://github.com/erh/viamrtsp. We will be deleting this repo in short order.

**Pull request closed without merging**